src/presenters/open-graph/image-presenter.php

```diff
@@ -10,6 +10,13 @@
  */
 class Image_Presenter extends Abstract_Indexable_Presenter {
 
+	/**
+	 * The tag key name.
+	 *
+	 * @var string
+	 */
+	protected $key = 'og:image';
```

Comment:
@increddibelly I believe this class property shouldn't be added here, as Image_Presenter doesn't extend the Abstract_Indexable_Tag_Presenter class.

Reply:
Maybe related to the above: 
The **og:image** field is also not added to the REST response in the `yoast_head_json` field, even though it exists in the `yoast_head` one

Reply:
I've made a ticket P2-1083 to deal with this class; it's got logic in its presenter which blocks the normal route of the tag presenter.

Reply:
actually Herre solved it just now in a far simpler way :-)